src/mixins/document-mixin.ts

```diff
@@ -2,11 +2,22 @@ import { AxiosResponse } from 'axios'
 import axios from '@/utils/axios-auth'
 import { Component, Vue } from 'vue-property-decorator'
 import { DocumentUpload } from '@/interfaces'
+import { PdfPageSize } from '@/enums'
+import pdfjsLib from 'pdfjs-dist/build/pdf'
+pdfjsLib.GlobalWorkerOptions.workerSrc = 'https://cdn.jsdelivr.net/npm/pdfjs-dist@2.0.943/build/pdf.worker.min.js'
 
 @Component({})
 export default class DocumentMixin extends Vue {
   readonly UPLOAD_FAILED_MESSAGE = 'An error occurred while uploading.  Please try again.'
   readonly MAX_FILE_SIZE = 10 * 1024 // 10 MB in KB
+  readonly pageSizeInfo = {
```

Comment:
I suggest naming this "pageSizeDict" to emphasize the fact that this is a dictionary (for future devs). Up to you.

Reply:
updated to use `pageSizeDict`